db/migrate/20130612104243_reporting_migrate_serialized_yaml.rb

```diff
@@ -17,16 +17,16 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #++
 
-require Rails.root.join("db","migrate","migration_utils","yaml_migrator").to_s
+require Rails.root.join("db","migrate","migration_utils","legacy_yamler").to_s
```

Comment:
Space missing after comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.